odxtools/endofpdufield.py

```diff
@@ -148,9 +141,14 @@ def _resolve_references(self, # type: ignore[override]
         """
         if self.structure_ref is not None:
             self._structure = odxlinks.resolve(self.structure_ref)
-        else:
+        elif self.structure_snref is not None:
             self._structure = parent_dl.data_object_properties[self.structure_snref]
 
+        if self.env_data_desc_ref is not None:
+            self._env_data_desc = odxlinks.resolve(self.env_data_desc_ref)
+        elif self.env_data_desc_snref is not None:
+            self._env_data_desc = parent_dl.data_object_properties[self.env_data_desc_snref]
```

Comment:
Does not looks related to PR topic, what this about?

Reply:
this was probably a "drive-by" change most likely motivated because I got some errors in the end-of-pdu-field class. I'll split it out.